stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+};
+
+export const withUserLink = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with user link example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link fd-feed-list__thumb"  
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)"
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">
+                        <a href="#" class="fd-link fd-link--emphasized" tabindex="0">Joe Doe: </a> 
+                    </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+withUserLink.storyName = 'With User link';
+withUserLink.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item with link'
+    }
+};
+
+export const showMoreButton = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with show more text example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus. Morbi tempor eu elit eu consequat. Sed at lorem a ex consequat pharetra. Etiam convallis odio at vulputate venenatis. In sit amet pharetra urna. Ut nulla nisi, porta at ligula a, elementum ullamcorper lectus. Suspendisse blandit, risus nec vestibulum volutpat, nulla neque rhoncus dolor, vitae rutrum lectus neque nec ligula. Maecenas pulvinar, ligula nec fringilla volutpat, urna quam rutrum tellus, vi...
+                    <a class="fd-link fd-feed-list__more" tabindex="0">More</a>        
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+showMoreButton.storyName = 'Show More Button';
+showMoreButton.parameters = {
+    docs: {
+        storyDescription: 'More Button'

```

Comment:
Feed list items can display a _MORE_ link that can show more text (when/if it is truncated). The link is subtly displayed in grey, and highlights when the user hovers over the feed. When hovering over the link, the text is highlighted and underlined.